src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -27,7 +28,6 @@ import {
     EvidenceType,
     IOncoKbData,
 } from 'cbioportal-frontend-commons';
-import { generateCopyNumberAlterationQuery } from 'cbioportal-frontend-commons/src/lib/oncokb/OncoKbUtils';
```

Comment:
these kinds of imports can still sneak in, but they are not as bad as import from `../../../packages`

Reply:
is there an easy way to avoid this? E.g. adding a grep for `cbioportal-frontend-commons/.*`? Or maybe a little more fancy with grepping all packages `PACKAGES=$(ls packages/* | cut -d/ -f2)` and then using that to grep

Reply:
we still need import statements like `import 'cbioportal-frontend-commons/dist/styles.css'`, so maybe we only allow importing style files but nothing else?